test/system.spec.ts

```diff
@@ -1,74 +1,230 @@
+          const fileName = faker.system.fileName();
+
+          expect(
+            fileName.startsWith('/'),
+            'generated fileNames should not have path separators'
+          ).toBeFalsy();
+        });
+
+        it('should return filenames with ext on the end', () => {
+          const fileName = faker.system.fileName();
+
+          expect(
+            fileName,
+            'generated fileNames should have an extension'
+          ).toContain('.');
+        });
+      });
+
+      describe('commonFileName()', () => {
+        it('should return common file name without system path separators', () => {
+          const fileName = faker.system.commonFileName();
+
+          expect(
+            fileName,
+            'generated common file name should not have path separators'
+          ).not.toContain('/');
+        });
+
+        it('should return common file name with ext on the end', () => {
+          const fileName = faker.system.commonFileName();
+
+          expect(
+            fileName,
+            'generated common file name should have a extension'
+          ).toContain('.');
+        });
+
+        it('should return common file name with given ext', () => {
+          const fileName = faker.system.commonFileName('txt');
+
+          expect(
+            fileName,
+            'generated common file name should not have path separators'
+          ).not.toContain('/');
+          expect(
+            fileName,
+            'generated common file name should have given ext'
+          ).toContain('txt');
+        });
+      });
+
+      describe('commonFileType()', () => {
+        it('should return common file types', () => {
+          const fileType = faker.system.commonFileType();
+          const fileTypes = ['application', 'audio', 'image', 'text', 'video'];
+
+          expect(
+            fileTypes,
+            `generated common file type should contain one of [${fileTypes.join(
+              ','
+            )}]. Got "${fileType}".`
+          ).toContain(fileType);
+        });
+      });
+
+      describe('commonFileExt()', () => {
+        it('should return common file types', () => {
+          const fileExt = faker.system.commonFileExt();
+          const extList = [
+            'pdf',
+            'mpeg',
+            'wav',
+            'png',
+            'jpeg',
+            'gif',
+            'mp4v',
+            'mpeg',
+            'htm',
+            'm2a',
+          ];
+
+          expect(
+            extList,
+            `generated common file ext should be one of [${extList.join(
+              ','
+            )}]. Got "${fileExt}".`
+          ).include(fileExt);
+        });
+      });
+
+      describe('mimeType()', () => {
+        it('should return mime types', () => {
+          const mimeType = faker.system.mimeType();
+
+          expect(
+            mimeType,
+            `generated mime types should be valid mime types.`
+          ).match(/.+\/*./gm);
```

Comment:
Is the pattern correct?
```suggestion
          ).match(/.+\/.+/gm);
```

Reply:
There is a `validator.isMimeType` :smiley: